sjb/config/common/test_cases/origin_release_install_azure_39.yml

```diff
@@ -163,6 +163,34 @@ extensions:
           -e "openshift_azure_input_image_prefix=centos7" \
           playbooks/azure/openshift-cluster/launch.yml
 
+    - type: "script"
+      title: "run the acs-engine e2e test"
+      script: |-
+        git clone -b custom-images-e2e https://github.com/kargakis/acs-engine.git
+        cd acs-engine
+
+        set -a +o xtrace
```

Comment:
below when ansible.sh calls entrypoint-provider doesn't it do this for you? why do you need it here?